docs/howto/push-notifications.md

```diff
@@ -170,6 +170,64 @@ idiosyncratic capitalization shown.
 
 ### Current workaround
 
+This workaround means using a development server. If that won't work for you,
+you'll have to try the second workaround, below.
+
+We can tell our development server to talk to Apple's APNs "sandbox" server,
```

Comment:
> *You can tell *your development server [...]

perhaps? (I'm not sure how it's relevant that we can tell CZO to do this.)

Reply:
Haha indeed; I've gotten so used to using "we," in commit messages, and this looks like spillover from that habit.